site/content/en/docs/tasks/run_jobsets.md

```diff
@@ -70,6 +70,10 @@ The first [PriorityClassName](https://kubernetes.io/docs/concepts/scheduling-evi
               priorityClassName: high-priority
 ```
 
+### d. Suspension
+
+You should create the JobSet in a suspended state, as Kueue will decide when it’s the best time to start the Job.
```

Comment:
This is not absolutely necessary, as the Kueue's webhook would set it https://github.com/kubernetes-sigs/kueue/blob/64d980c312b600d18a0a63988666b6195a83ac58/pkg/controller/jobs/jobset/jobset_webhook.go#L71